language-support/hs/bindings/src/DA/Ledger/Convert.hs

```diff
@@ -0,0 +1,101 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+
+-- Convert between HL Ledger.Types and the LL types generated from .proto files
+module DA.Ledger.Convert (lowerCommands) where
+
+import           Data.Maybe         (fromMaybe)
+import           Data.Vector        as Vector (fromList)
+
+import qualified DA.Ledger.LowLevel as LL
+import           DA.Ledger.Types
+
+xxx :: a -- TODO: not finished while any instances of "xxx" of "_" remain in this file
```

Comment:
I just write undefined - you're also not finished if undefined exists in your code...